doc/source/reliability/innodb_corrupt_table_action.rst

```diff
@@ -23,8 +23,7 @@ running in the background.
 
 If the :variable:`innodb_force_recovery` variable is set to a low value
 and you expect the server to crash,
```

Comment:
with innodb_force_recovery set to low value, the intention is to not crash. Infact provide levels of recoverability after a crash and startup failures.

Please check this once: https://dev.mysql.com/doc/refman/8.0/en/forcing-innodb-recovery.html